mrtt-ui/src/components/CheckboxGroup/CheckboxGroup.js

```diff
@@ -1,36 +1,41 @@
-import { Checkbox, FormControlLabel, FormGroup, TextField } from '@mui/material'
+import { Checkbox, FormControlLabel, FormGroup, Stack, TextField } from '@mui/material'
 import PropTypes from 'prop-types'
 import { forwardRef, useEffect, useState } from 'react'
 import language from '../../language'
+import { makeValidClassName } from '../../library/strings/makeValidClassName'
 
-const getValueToReturn = ({ nonOtherSelectedValues, otherInputValue, isOtherChecked }) => {
-  const valueToReturn = { selectedValues: nonOtherSelectedValues ?? [] }
+const getValueToReturn = ({ nonOtherSelectedValues, otherInputValue, isOtherChecked, event }) => {
+  const valueToReturn = { selectedValues: nonOtherSelectedValues ?? [], event }
```

Comment:
adding original event to object being returned so we can grab its value for the resetting of nested answers when an item is unchecked (because the data format required the option value). see `handleAimChangeAndResetStakeholdersIfAppropriate`